src/Blocks/custom/carousel/carousel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Template for the Carousel Block.
+ *
+ * @package Unicorns
+ */
+
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$blockJsClass = Components::checkAttr('blockJsClass', $attributes, $manifest);
+$isLoop = Components::checkAttr('isLoop', $attributes, $manifest);
+$showItems = Components::checkAttr('showItems', $attributes, $manifest);
+
+$carouselClass = Components::classnames([
+	$blockClass,
+	$blockJsClass,
+	'swiper-container',
+]);
+
+?>
+
+<div class="<?php echo esc_attr($carouselClass); ?>" 
+	data-swiper-loop="<?php echo esc_attr($isLoop); ?>" 
+	data-show-items="<?php echo esc_attr($showItems); ?>">
+
+	<div class="<?php echo esc_attr('swiper-wrapper'); ?>">
+		<?php
+		$innerBlockContent = str_replace( "{$blockClass}__item", "{$blockClass}__item {$blockJsClass}-item", $innerBlockContent);
+		$innerBlockContent = str_replace( "{$blockJsClass}-item-inner", "{$blockClass}__item-inner {$blockJsClass}-item-inner", $innerBlockContent);
```

Comment:
never ever do this you can't be sure that this will not break anything else

Reply:
Same question as in other PR - should i fix this with the `render_block_data` filter or change the Image component to allow passing extra classes that will create selectors?